browsergym/core/src/browsergym/core/action/functions.py

```diff
@@ -14,6 +14,7 @@
 send_message_to_user: callable = None
 report_infeasible_instructions: callable = None
 demo_mode: Literal["off", "default", "all_blue", "only_visible_elements"] = None
+force_actions: bool = False
```

Comment:
I'd change the name for something more explicit, like `retry_with_force`. From `force_actions` one could think all actions are always executed with `force=True`.

Reply:
agreed